scripts/update_boot_iso

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import shutil
+import time
+import sys
+import subprocess
+
+# Absolute path to the main project directory
+PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Relative path to the ISO folder within the project
+ISO_FOLDER = os.path.join(PROJECT_DIR, "result", "iso")
+
+# Initial boot ISO we will update
+INPUT_ISO = os.path.join(ISO_FOLDER, "boot.iso")
+INPUT_ISO_REVISION_FILE = os.path.join(ISO_FOLDER, "boot.iso.git_rev")
+
+# Updated boot ISO (including Anaconda updates image and possibly other bits)
+UPDATED_ISO = os.path.join(ISO_FOLDER, "updated_boot.iso")
+
+# Folder needed to include updates image into the updated ISO
+UPDATES_FOLDER = os.path.join(ISO_FOLDER, "images")
+UPDATES_IMAGE = "updates.img"
+BOOT_OPTIONS = "inst.sshd inst.nokill"
```

Comment:
I know this may be obvious to you and other folks that work on anaconda for a long time, but do you think it would make sense if we could add a link for the docs that explain the command here? Like: 


```suggestion
# For inst.sshd = https://anaconda-installer.readthedocs.io/en/latest/boot-options.html#inst-sshd
# For inst.nokill = https://anaconda-installer.readthedocs.io/en/latest/boot-options.html#inst-nokill
BOOT_OPTIONS = "inst.sshd inst.nokill"
```

Or, if that's a lot of text, we could just simply: 


```suggestion
More on the boot options: https://anaconda-installer.readthedocs.io/en/latest/boot-options.html#anaconda-boot-options
BOOT_OPTIONS = "inst.sshd inst.nokill"
```

Reply:
I think we can do the more verbose version to make sure its users are not confused. :)